### ISSUE TITLE
403 Forbidden Error

### ISSUE THREAD

**Issue by krampfstadtstudio** [7x11x13/bandcamp-auto-uploader#4]
it worked just fine 2 days ago and now it doesnt...

![Untitled](https://github.com/user-attachments/assets/65681beb-4aea-46d5-9732-cb45d6c05631)

**Comment by 7x11x13** [comment]
Is the cookies.txt file up to date? Maybe the cookies have changed since you used it before?

**Comment by krampfstadtstudio** [comment]
i copied new cookies from both my artist page and main bandcamp page using Flag Cookies Chrome extension, but it didnt help
________________________________________________________________________Danilo Crnogorac
www.imdb.com/name/nm5592832/www.linkedin.com/in/danilocrnogoracwww.soundkrampf.blogspot.com
 

    On Thursday, July 18, 2024 at 04:50:05 PM GMT+2, 7x11x13 ***@***.***> wrote:  
 
 


Is the cookies.txt file up to date? Maybe the cookies have changed since you used it before?

**Comment by 7x11x13** [comment]
Well, I'm not able to reproduce the issue. I just pushed a new update to add more debugging statements. Can you try this version (https://github.com/7x11x13/bandcamp-auto-uploader/releases/tag/v0.2.10), but before you run it, set debug to true in your config file? The config file location is printed on the first line of output in green as you can see above. Just change the line `"debug": false` to `"debug": true` to enable debugging. Then when you run it, it will give more logs than before, and you can post them here.

**Comment by krampfstadtstudio** [comment]
it does not work again. and i dont see that 'debug' line anywhere...

**Comment by 7x11x13** [comment]
Can you add it to the config then? Just put a comma after the last line before the ending `}` and write `"debug": true`

**Comment by krampfstadtstudio** [comment]
![image](https://github.com/user-attachments/assets/079de4a8-00c9-4c00-b43c-b087c07d118f)
 like this?

**Comment by 7x11x13** [comment]
No, open the file `C:/Users/Win10/AppData/Local/7x11x13/bandcamp-auto-uploader/config.json` and put it in there

**Comment by krampfstadtstudio** [comment]
i dont have that file. i just have bandcamp-auto-uploader-win.exe

**Comment by 7x11x13** [comment]
From your screenshot above it says
![image](https://github.com/user-attachments/assets/2ca87708-00e7-4553-8235-a6f5953bdc3e)
surely it must exist?

**Comment by krampfstadtstudio** [comment]
![image](https://github.com/user-attachments/assets/d88dab1c-d464-42a7-8939-0988be53955a)

**Comment by 7x11x13** [comment]
Just to confirm, you're able to visit bandcamp.com in your regular browser with no error? Also, can you try deleting every line from the cookies.txt file except the one containing the line with `identity` in it, and lines starting with `#`? So your new cookies.txt file would look like:
```
# Netscape HTTP Cookie File
# https://curl.haxx.se/rfc/cookie_spec.html
# This is a generated file! Do not edit.

.bandcamp.com	TRUE	/	TRUE	1939232518	identity     ...
```

**Comment by krampfstadtstudio** [comment]
i can access it with no error. i changed the cookies file as you said

![image](https://github.com/user-attachments/assets/68199ea5-590d-478b-887f-28efeec77655)

but it is not working

**Comment by 7x11x13** [comment]
I meant, keep the cookies.txt file you already had, but delete the lines that don't start with `#` and also don't contain `identity`, so the result would look like that, except there would be a long string of characters instead of `...`

**Comment by krampfstadtstudio** [comment]
still not working

![image](https://github.com/user-attachments/assets/62d93ff1-68bc-47a6-a138-4b4f5cf6c464)

![image](https://github.com/user-attachments/assets/f764fbb8-72e9-40f9-98b5-f675d570f772)

**Comment by 7x11x13** [comment]
Well I can't really think of what the problem might be, and I can't reproduce it to test myself. I can only think maybe you're being ratelimited for trying to upload too much and maybe it will work if you try again later... for now it seems like a dead end

**Comment by krampfstadtstudio** [comment]
How can i reset the program? To see what i saw the first time i opened it, like i want to login for the first time


_______________________________________________________________________

Danilo Crnogorac


www.imdb.com/name/nm5592832/

www.linkedin.com/in/danilocrnogorac

www.soundkrampf.blogspot.com 
 
  On Fri, Jul 19, 2024 at 5:59 AM, ***@***.***> wrote:   


Well I can't really think of what the problem might be, and I can't reproduce it to test myself. I can only think maybe you're being ratelimited for trying to upload too much and maybe it will work if you try again later... for now it seems like a dead end

**Comment by 7x11x13** [comment]
> How can i reset the program? To see what i saw the first time i opened it, like i want to login for the first time

You can reset by deleting the config.json file which I mentioned previously

**Comment by 7x11x13** [comment]
Hey, I believe I fixed this issue in v0.2.11 if you want to try again
